core/Updates/2.13.1.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+
+namespace Piwik\Updates;
```

Comment:
I placed this update file first in UsersManager plugin since it is needed because of logic there and couldn't really decide whether to place it in that plugin or in core. Placed it in core since it is kinda nice to be able to see all update scripts in one place but I think it rather belongs to UsersManager and we should maybe start putting them into the related plugins if they belong there.